pkg/core/bundle_pack.go

```diff
@@ -313,8 +314,18 @@ func uploadBundle(ctx context.Context, bundle *Bundle, bundleEntriesPerFile uint
 	return nil
 }
 
-func uploadBundleDescriptor(ctx context.Context, bundle *Bundle) error {
+func validateBundle(bundle *Bundle) bool {
+	if bundle.BundleDescriptor.Deduplication == "" {
```

Comment:
this is breaking again against existing stuff, without obvious necessity. Why don't we assume Blake is the default?

Reply:
This would be breaking if the master was to be used against existing stuff. This is to insure that future code paths do not forget to set it.